internal/helpers/rego/config/contexts.rego

```diff
@@ -0,0 +1,70 @@
+package circleci.config
+
+import data.circleci.utils
+
+import future.keywords
+
+contexts_blocked_by_project_ids(project_id, context_list) = {reason |
```

Comment:
Can we add a comment explaining what this policy intends to do, just to be more explicit?